contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -40,9 +38,14 @@ end
 func deposit_handled(l2_token : felt, account : felt, amount : Uint256):
 end
 
+@event
+func rewards_minted(l2_reward_token : felt, account : felt, amount : Uint256):
+end
+
 @event
 func bridged_rewards(l2_token : felt, acocunt : felt, amount : Uint256):
```

Comment:
Can we make these two events grammatically consistent? Either `minted_rewards` or `rewards_bridged`?

Reply:
And take into account `deposit_handled`